proto/stride/records/records.proto

```diff
@@ -72,6 +76,19 @@ message HostZoneUnbonding {
     (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
     (gogoproto.nullable) = false
   ];
+  string st_tokens_to_burn = 8 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.nullable) = false
+  ];
+  string native_tokens_to_unbond = 9 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.nullable) = false
+  ];
+  string claimable_native_tokens = 10 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.nullable) = false
+  ];
+  uint64 undelegation_txs_in_progress = 11;
```

Comment:
Just curious, do we need this variable if we're already storing `native_tokens_to_unbond` and `claimable_native_tokens` / `unbonding_time`? 

Even if it's not used, seems reasonable to me to store it for inspectability!

Reply:
Yeah so this was for an edge case that came up kind of late. The idea is if one of the undelegation batches lands on the host zone in time, but we don't get the ack back until after the epoch, we need to make sure not to resubmit the  undelegation for that batch. 

So, although I don't love adding this field, I think adding a semaphore like this is the easiest way to build a locking mechanism to make sure everything comes back before we retry. Open to other ideas if you have em though!

Reply:
Example:
* EUR with 1000 tokens
* Submit two batches, first for 400, then for 600
* First batch lands and acks, we decrement the EUR to 600
* Next batch lands but ack is still pending
* Next epoch kicks off, we need to make sure we don't submit an undelegation for 600 again